src/test/kotlin/blackjack/model/DealerTest.kt

```diff
@@ -121,5 +128,4 @@ class DealerTest {
         assertThat(player.gameResults.wins).isEqualTo(1)
         assertThat(dealer.gameResults.loses).isEqualTo(1)
     }
-
 }
```

Comment:
<img width="1552" height="987" alt="image" src="https://github.com/user-attachments/assets/2b6962f3-44b7-48c0-ac73-cf7d2fe8a62c" />

Could you check all the tests again?
It looks like there are some compile errors 👀